unstructured/metrics/text_extraction.py

```diff
@@ -0,0 +1,52 @@
+from typing import Tuple
+
+from rapidfuzz.distance import Levenshtein
+
+
+def calculate_edit_distance(
+    output: str,
+    source: str,
+    weights: Tuple[int, int, int] = (2, 1, 1),
+    return_as: str = "score",
+) -> float:
+    """
+    Calculates edit distance using Levenshtein distance between two strings.
+
+    Args:
+        output (str): The target string to be compared.
+        source (str): The reference string against which 'output' is compared.
+        weights (Tuple[int, int, int], optional): A tuple containing weights
+            for insertion, deletion, and substitution operations in the edit
+            distance calculation. Default is (2, 1, 1).
+        return_as (str, optional): The type of result to return, one of
+            ["score",, "distance"].
+            Default is "score".
+
+    Returns:
+        float: The calculated edit distance or similarity score between
+            the 'output' and 'source' strings.
+
+    Raises:
+        ValueError: If 'return_as' is not one of the valid return types
+        ["score", "distance"].
+
+    Note:
+        This function calculates the edit distance (or similarity score) between
+        two strings using the Levenshtein distance algorithm. The 'weights' parameter
+        allows customizing the cost of insertion, deletion, and substitution
+        operations. The 'return_as' parameter determines the type of result to return:
+        - "score": Returns the similarity score, where 1.0 indicates a perfect match.
```

Comment:
Looks great! Could you update the summary to reflect that we only offer 2 return types now?